scripts/uberenv/packages/mfem/package.py

```diff
@@ -479,10 +479,10 @@ def find_optional_library(name, prefix):
                 ld_flags_from_library_list(spec[sun_spec].libs)]
 
         if '+petsc' in spec:
+            # SERAC EDIT BEGIN (TODO: spack PR)
             options += [
-                'PETSC_OPT=%s' % spec['petsc'].headers.cpp_flags,
-                'PETSC_LIB=%s' %
-                ld_flags_from_library_list(spec['petsc'].libs)]
+                'PETSC_DIR=%s' % spec['petsc'].prefix]
```

Comment:
I guess this begs the question why the rest of the dependencies build the link and include lines if you can just override the <tpl name>_DIR makefile variable

Reply:
I think building the link/include stuff in spack is preferable in case the directory structure assumed in the makefile doesn't match what was installed - it's just in this case that no other option is available (for static linking.)